### PR TITLE
tests: Test drive exceeded time behavior

### PR DESCRIPTION
**Pull request by nirs**
The way we set and clear exceeded time is little too delicate. Lets
protect with it with a test.

The new test added as simple function since the relevant class was was
not converted properly to pytest, and we cannot use pytest fixtures in
the class. Converting the module to pytest properly will have to be done
later.

Completes #80.

Signed-off-by: Nir Soffer <nsoffer@redhat.com>

### CODE REVIEW

File: tests/virt/vmstorage_test.py [hunk 1]
```diff
@@ -617,6 +621,27 @@ def test_block_threshold_stale_path(self):
...
+    drive = Drive(log, **conf)
+
+    # Exceeded time not set yet.
+    assert drive.exceeded_time is None
+
+    # Setting threshold state does not set exceeded time.
+    drive.threshold_state = BLOCK_THRESHOLD.SET
+    assert drive.exceeded_time is None
+
+    # Getting threshold event set exceeded time.
```
Comment (barpavel) [inline tests/virt/vmstorage_test.py:635]: `set` --> `sets`

File: tests/virt/vmstorage_test.py [hunk 2]
```diff
@@ -617,6 +621,27 @@ def test_block_threshold_stale_path(self):
...
+    monkeypatch.setattr(time, "monotonic_time", lambda: 123.0)
+    drive.on_block_threshold("/path")
+    assert drive.exceeded_time == 123.0
+
+    # Chanding threshold clears exceeded time.
```
Comment (barpavel) [inline tests/virt/vmstorage_test.py:640]: `Chanding` --> `Changing`.

**Review by vjuranek — COMMENTED** [review]
LGTM but would be nice to fix the tipos found by Pavel

**Comment by nirs** [comment]
@barpavel @vjuranek thanks for reviewing, typos fixed in new version.

**Pull request merged**